jscomp/core/lam_pass_remove_alias.ml

```diff
@@ -39,7 +39,11 @@ let simplify_alias
     | Lvar v ->
       begin match (Ident_hashtbl.find_opt meta.alias_tbl v) with
       | None -> lam
-      | Some v -> Lam.var v 
+      | Some v ->
+        if Ident.persistent v then 
+          Lam.prim ~primitive:(Pgetglobal v) ~args:[] Location.none
+        else 
+         Lam.var v 
         (* This is wrong
```

Comment:
bug fix: the alias table may contain global variables, we should recover here to avoid 
`Lglobal v` leaking into `Lvar v`
TODO: we should remove Pgetglobal as Lglobal instead